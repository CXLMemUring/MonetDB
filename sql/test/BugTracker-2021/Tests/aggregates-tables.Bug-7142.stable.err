stderr of test 'aggregates-tables.Bug-7142` in directory 'sql/test/BugTracker-2021` itself:


# 11:30:33 >  
# 11:30:33 >  "mclient" "-lsql" "-ftest" "-tnone" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-218832" "--port=38265"
# 11:30:33 >  

MAPI  = (monetdb) /var/tmp/mtest-218832/.s.monetdb.38265
QUERY = CREATE AGGREGATE linear_least_squares(xg DOUBLE, yg DOUBLE)
        RETURNS TABLE (a DOUBLE, b DOUBLE)
        LANGUAGE PYTHON {
            import numpy as np
            # Define an inner function to do the work
            def perform_least_squares(x, y):
                mx = np.average(x)
                my = np.average(y)
                N = len(x)
                a = (np.dot(x,y) - mx*my)/(np.dot(x,x) - N*mx*mx)
                b = (my*np.dot(x,x) - mx*(np.dot(x,y)))/(np.dot(x,x) - N*mx*mx)
                return [a, b]
            ab = {"a": list(), "b": list()}
            try:
                groups = np.unique(aggr_group)
ERROR = !CREATE AGGREGATE: aggregates cannot return tables
CODE  = 42000

# 11:30:33 >  
# 11:30:33 >  "Done."
# 11:30:33 >  

