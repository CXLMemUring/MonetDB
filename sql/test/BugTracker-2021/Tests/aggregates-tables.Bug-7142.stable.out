stdout of test 'aggregates-tables.Bug-7142` in directory 'sql/test/BugTracker-2021` itself:


# 11:30:33 >  
# 11:30:33 >  "mclient" "-lsql" "-ftest" "-tnone" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-218832" "--port=38265"
# 11:30:33 >  

#START TRANSACTION;
#ROLLBACK;

# 11:30:33 >  
# 11:30:33 >  "Done."
# 11:30:33 >  

